### DISCUSSION TITLE
Night Light

### DISCUSSION THREAD

**Discussion by Newer1107** [prasanthrangan/hyprdots#240] [Ideas]
wl-gammarelay seems fine for night light and also has waybar integration examples on github, doesn't take much to setup either.
can we please add this?

source:
(https://www.reddit.com/r/hyprland/comments/12qczxw/comment/jgs7mrj/?utm_source=share&utm_medium=web2x&context=3)
(https://github.com/jeremija/wl-gammarelay)

**Comment by prasanthrangan** [comment]
cool thanks, will look into this!
